pyfr/backends/openmp/base.py

```diff
@@ -50,6 +49,8 @@ def __init__(self, cfg):
             except:
                 pass
 
+        self._providers.append(gimmik.OpenMPGiMMiKKernels(self))
```

Comment:
This works but now blas is also prioritized over gimmik. Do you have a better suggestion?